src/app/dim-ui/CollapsibleTitle.scss

```diff
@@ -14,10 +14,13 @@
   align-items: center;
   cursor: pointer;
 
-  &:hover,
+  &:hover {
+    background-color: rgba(0, 0, 0, 0.4);
+    color: var(--theme-accent-primary);
+  }
+
   &.collapsed {
     background-color: rgba(0, 0, 0, 0.4);
-    color: #ccc;
```

Comment:
I can't actually recall if this was intentional — it may have related to the changes when I was experimenting with a light theme, but given how subtle it is i'll leave this change in